### PR TITLE
Remove redundant logic in `invoke()`

### PR DESCRIPTION
**Pull request by tslmy**
Also clarified things quite a bit.

### CODE REVIEW

File: imhotep/app.py [hunk 1]
```diff
@@ -180,33 +180,31 @@ def invoke(
...
-                matching_numbers = set(added_lines).intersection(violating_lines)
+                matching_numbers = violating_lines.intersection(pos_map)
                 for i in matching_numbers:
                     error_count += 1
                     if error_count > max_errors:
                         continue
                     reporter.report_line(
                         cinfo.origin,
                         entry.result_filename,
-                        x,
```
Comment (justinabrahms) [inline imhotep/app.py:209]: I believe this is a backwards incompatible change w/ the other reporters, no?

**Review by justinabrahms — CHANGES_REQUESTED** [review]

**Comment by tslmy** [comment]
Are there other reporters?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by justinabrahms** [comment]
Thought so, but I don't see them. Maybe I never released them. Will give this another review pass this evening.

**Comment by tslmy** [comment]
Cool. Just let me know.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Review by justinabrahms — APPROVED** [review]

**Pull request merged**